services/network/public/mojom/content_security_policy.mojom

```diff
@@ -124,6 +124,7 @@ enum CSPDirectiveName {
   TrustedTypes,
   UpgradeInsecureRequests,
   WorkerSrc,
+  [EnableIf=cobalt_custom_csp]H5vccLocationSrc,
```

Comment:
If we do add alternative naming to this, as in `cobalt-location-src`, i'd suggest renaming the constant as well. `h5vcc` is incredibly obscure to unsuspecting reader of the code.